DigitalLearningSolutions.Data.Migrations/202205131200_LiftConstraintsOnDeprecatedColumns.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202205131200)]
+    public class LiftConstraintsOnDeprecatedColumns : Migration
+    {
+        public override void Up()
+        {
+            Delete.DefaultConstraint().OnTable("AdminAccounts").OnColumn("CategoryID");
+
+            Alter.Table("AdminAccounts").AlterColumn("Password_deprecated").AsString(250).Nullable();
+            Alter.Table("AdminAccounts").AlterColumn("EITSProfile_deprecated").AsCustom("varchar(max)").Nullable();

```

Comment:
I don't think this covers all the columns we'd expect to be changed here. I would expect at least Forename_deprecated and Surname_deprecated to also have had their 'not null' removed. Adding these would cover the empty strings that have been added in [HEEDLS-856](https://github.com/TechnologyEnhancedLearning/DLSV2/pull/1110/files/eee2749f83ddbeba19fb0e1e003159ab30c806aa#r872137481)

Reply:
This is an oversight on my part - those empty strings should all be removed as redundant once this migration is done. I'll do that in this branch for the delegate account fields.

Reply:
It sounds like 856 is a lot closer to being merged, so I guess the admin one will end up being sorted here